com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Platform.java

```diff
@@ -224,6 +224,9 @@ public boolean matchesPair(String pair) {
     }
 
     public boolean matchesOS(PlatformProfile.OS os) {
+        if (os.equals(OS.OS_ID_GENERIC)) {
+            return true;
+        }
```

Comment:
make sure that generic option in the texture profile matches all the OSes
![CleanShot 2025-02-13 at 14 59 14@2x](https://github.com/user-attachments/assets/7778d723-fab1-4af1-8a1e-95c2dd192027)